src/components/CardSeparator.tsx

```diff
@@ -26,9 +27,9 @@ const getTailwindGradient = (gradient: Gradient) =>
       : undefined
   )
 
-const connectiveBlock = (vertical = true) =>
+const connectiveBlock = (vertical?: boolean, className?: TArg) =>
```

Comment:
Can we limit this `className` to `display` function only?

Reply:
Actually, do we use it?

Reply:
Removed it.